lsp/lsp-harness/tests/inputs/goto-basic.ncl

```diff
@@ -0,0 +1,8 @@
+### /goto.ncl
```

Comment:
Just for the record, the reason I went for `###` as a marker was because (unlike the nickel tests) the marker lines don't occur only at the beginning of the file. I didn't want to forbid the nickel content from containing comments, but I figured it was fine to insist that none of those comments start with `###`...

Reply:
I see. Anyway that's a detail that can be changed easily later on, and it's not user-facing. We maybe we want to have comments in LSP tests, maybe not :man_shrugging: for now, as long as the syntax is documented somewhere, I think we can proceed as it is